pkg/oc/admin/prune/images.go

```diff
@@ -412,6 +417,10 @@ func (o PruneImagesOptions) Run() error {
 		fmt.Fprintln(o.ErrOut, "Dry run enabled - no modifications will be made. Add --confirm to remove images")
 	}
 
+	if o.PruneRegistry != nil && !*o.PruneRegistry {
+		fmt.Fprintln(o.Out, "Only API objects will be removed.  No modifications to the image registry will be made.")
```

Comment:
Please add a warning that re-doing the pruning without this option (or with `--prune-registry=true`) will not lead to pruning the storage of image registry.

Reply:
/hold

Reply:
At this point it's too late to show a warning for the first time, the user won't be asked for any additional confirmations. Should we add this warning to help?

Reply:
There were previous comments around keeping the help text concise, so that gives me pause, but yeah if we are going to warn, I see no other place to put it where the warning would do some good.

Reply:
In fact, I previously had a reference to hard prune in the help text before and @bparees had me remove it.

Reply:
Well that was prior to my understanding that once you run this, hard prune is the only way to finish cleaning up the registry.  I'm ok w/ a concise warning in the help text that references the need to run hard prune (not regular prune) to clean up the registry after doing this.

Reply:
warning to help added and pushed